doc/internal/man3/ossl_provider_new.pod

```diff
@@ -235,9 +235,9 @@ is returned.
 
 ossl_provider_teardown() doesnt't return any value.
 
-ossl_provider_get_param_types() returns a pointer to an I<OSSL_ITEM>
-array if this function is available in the provider, otherwise
-NULL.
+ossl_provider_get_param_types() returns a pointer to a constant
```

Comment:
Is it worth highlighting the _constant_ here?  It's a critical part of the call.

Reply:
Why?  They'll definitely notice if they try to access the returned value as non-const...  no?